odc_k8s/README.md

```diff
@@ -162,6 +174,8 @@ Copy the example to create your own live repo to setup ODC infrastructure to run
 | flux_additional_args | Use additional arg for connect flux to fluxcloud. Syntext: --connect=ws://fluxcloud | string | "" | No | 
 | flux_registry_exclude_images | comma separated string lists of registry images to exclud from flux auto release: docker.io/*,index.docker.io/* | string | "" | No | 
 | flux_helm_operator_version | Flux helm-operator release version | string | "1.0.1" | No | 
+| flux_registry_ecr | Use flux_registry_ecr for fluxcd ecr configuration | object({region=string includeId=string excludeId=string}) | { region="" includeId="" excludeId="602401143452" } | No | 
+| flux_service_account_arn | provide flux OIDC service account role arn | "" | No | 
```

Comment:
I would suggest that there should be separate PR on how to configure kubernetes service account in a live example repo. I am help you setup if you are blocked.

Reply:
Yeah, live example would be good, to go with the module. Specially since we will deprecate kube2iam use in due course.

I'm not blocked, everything is working for me.